x-pack/plugins/apm/public/components/shared/license_prompt/index.tsx

```diff
@@ -8,41 +8,29 @@
 import { EuiButton, EuiCard, EuiTextColor } from '@elastic/eui';
 import React from 'react';
 import { i18n } from '@kbn/i18n';
+import { useApmPluginContext } from '../../../context/apm_plugin/use_apm_plugin_context';
 import { useKibanaUrl } from '../../../hooks/use_kibana_url';
 
 export interface LicensePromptProps {
   text: string;
-  showBetaBadge?: boolean;
 }
 
-export function LicensePrompt({
-  text,
-  showBetaBadge = false,
```

Comment:
Is this change intended? Not beta anymore?

Reply:
Yes, I think was a left over